pkg/jetcloud/project.go

```diff
@@ -121,3 +131,22 @@ func (c *Client) configPath(wd string) string {
 func (c *Client) removeConfig(wd string) error {
 	return os.Remove(c.configPath(wd))
 }
+
+func (c *Client) confirmProjectInit(ctx context.Context, args InitProjectArgs) error {
+	tok := args.Token
+	member, err := c.GetMember(ctx, tok, tok.IDClaims().Subject)
+	if err != nil {
+		return err
+	}
+	fmt.Fprintf(
+		args.Stderr,
+		"Initializing project for %s. Enter y/yes to continue\n",
+		member.Organization.Name,
+	)
+	result := ""
+	fmt.Scanln(&result)
+	if result != "y" && result != "yes" {
```

Comment:
Can we also check `Y` and `Yes` and `YES`?

I also feel that ENTER should be treated as a yes.

Reply:
These prompts are usually of the form:
`Initializing project for %s. Do you want to continue? [Y/n]\n`
https://askubuntu.com/a/322601

which signifies that hitting ENTER will be treated as a yes, alongside uppercase and lowercase `y`